packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -47,6 +47,27 @@ const WorkflowStepStore = types
       }
 
       return false
+    },
+
+    findTasksByType (type) {
+      const tasksByType = Array.from(self.steps).map(([stepKey, step]) => {
+        if (step?.tasks) {
+          return Object.values(step.tasks).filter(task => {
```

Comment:
`step.tasks` is an array so I think you can filter that directly here.

Reply:
Ah, yes it's an object on the workflow.